### PR TITLE
Mepts 297 - DSD Indicator N8

### PR DESCRIPTION
**Pull request by moshonk**
Added the new indicator 

> “Number of active patients on ART (Non-pregnant and Non-Breastfeeding not on TB treatment) who participate in at least one measured DSD model (GAAC, AF, CA, DT, FR, DC, DS)”

 to the IM-ER Report

### CODE REVIEW

**Comment by cfaife** [comment]
it looks like the build is failing

**Comment by cfaife** [comment]
plase resolve the issue with `continuous-integration/travis-ci`

**Comment by moshonk** [comment]
@cfaife Issues resolved.

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java [hunk 1]
```diff
@@ -1505,6 +1559,115 @@ public CohortDefinition getAllPatientsOnSarcomaKarposi() {
...
+   * */
+  public CohortDefinition getPatientsParticipatingInAtLeastOneDsdModel() {
+    SqlCohortDefinition cd = new SqlCohortDefinition();
+    cd.setName("participatingInDsdModel");
+    cd.addParameter(new Parameter("startDate", "Start Date", Date.class));
+    cd.addParameter(new Parameter("endDate", "End Date", Date.class));
+    cd.addParameter(new Parameter("location", "Location", Location.class));
+    cd.setQuery(
+        DsdQueries.getPatientsParticipatingInDsdModel(
+            hivMetadata.getPrevencaoPositivaInicialEncounterType().getEncounterTypeId(),
```
Comment (cfaife) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java]: on my understanding it  looks like we are not  using the correct (encounter_type =6)  we are referring 34 and  35  while  the spec refers to  6

**Review by cfaife — CHANGES_REQUESTED** [review]

**Comment by moshonk** [comment]
Hi @cfaife  Please review when you get a chance. I am having to deal with lots of merge conflicts every time another PR is merged

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java (hunk 1 above)
Comment (moshonk) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java]: Fixed

**Pull request merged**